packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -890,6 +890,13 @@ class _CupertinoNavigationBarState extends State<CupertinoNavigationBar> {
 /// For advanced uses, an optional [middle] widget can be supplied to show a
 /// different widget in the middle of the navigation bar when the sliver is collapsed.
 ///
+/// This navigation bar is expanded only in portrait orientation. In landscape
+/// mode, the navigation bar remains collapsed. When collapsed (either by
+/// scrolling in portrait or by being in landscape), the [middle] widget is
```

Comment:
I think this is re-explaining the `middle` logic explained above in a different way, which is confusing. I think you should instead expand one or both of the previous paragraphs to explain only the collapsed view will show in landscape mode, imo.